src/cuckoocache.h

```diff
@@ -226,7 +223,7 @@ class cache
      *  [0,1) and simply multiply it by the size.  Then we just shift the result down by
      *  32-bits to get our bucket number.  The result has non-uniformity the same as a
      *  mod, but it is much faster to compute. More about this technique can be found at
-     *  http://lemire.me/blog/2016/06/27/a-fast-alternative-to-the-modulo-reduction/
+     *  http://lemire.me/blog/2016/06/27/a-fast-alternative-to-the-modulo-reduction/.
```

Comment:
generally best practice to have a space between url and period (or no period at all)